### DISCUSSION TITLE
I am trying to install superbase from svelteKit

### DISCUSSION THREAD

**Discussion by PaulAyling** [supabase/supabase#5677] [Questions]
I am using the instructions found in https://supabase.com/docs/guides/with-svelte but instead if using the svlete temple
tutorial](
I have successfully run superbase with svelte using this [[tutorial](https://supabase.com/docs/guides/with-svelte). However I really need the sveltkit rather than basing it on the [svelte template](https://svelte.dev/blog/the-easiest-way-to-get-started) . When I try to use superbase based on a sveltkit project I get the following error

11:50:01 [vite] Error when evaluating SSR module /src/superbaseClient.js:
ReferenceError: __api is not defined
    at /src/superbaseClient.js:3:21
    at async instantiateModule (/Users/bob/Documents/dev/javascript/svelte/svelteBp/node_modules/vite/dist/node/chunks/dep-971d9e33.js:56177:9)
11:50:01 [vite] Error when evaluating SSR module /src/routes/sbase.svelte:
ReferenceError: __api is not defined
    at /src/superbaseClient.js:3:21
    at async instantiateModule (/Users/bob/Documents/dev/javascript/svelte/svelteBp/node_modules/vite/dist/node/chunks/dep-971d9e33.js:56177:9)
__api is not defined
ReferenceError: __api is not defined
    at /src/superbaseClient.js:3:21
    at async instantiateModule (/Users/bob/Documents/dev/javascript/svelte/svelteBp/node_modules/vite/dist/node/chunks/dep-971d9e33.js:56177:9)

I have tried

1.     Created a new sveltkit project using the [sveltkit getting started instructions](https://kit.svelte.dev/)
2. Followed the getting started instructions in [superbase](https://supabase.com/docs/guides/with-svelte) but replaced the use of the svelte template with a sveltekit project.
3. Installed superbase with https://github.com/supabase-community/svelte-supabase
4.  I ignored the lines about installing the rollup plugins in the guide because I believe they are covered off by svelte kit.

**Comment by GaryAustin1** [comment]
You might look at this: https://github.com/supabase-community/auth-helpers/tree/main/examples/sveltekit
Especially if doing serverside stuff.
